### ISSUE TITLE
Update nested language proposal

### ISSUE THREAD

**Issue by jayvdb** [coala/projects#547]
The examples are confusing rather than helpful.

Php/HTML is a good example.

Another great example is reStructuredText, which allows specification of a language for a code block.  And Markdown now has a similar concept.  Our rstcheck bear already uses this and lints the code blocks.  Ideally this project would re-implement the rstcheck's linting of code blocks, so the splitting and spawning of linters is done inside coala instead of inside rstcheck, and the same functionality would be available for markdown without a custom bear.

**Comment by jayvdb** [comment]
Another example is IPython notebooks, which contain e.g. python code.

See our `PEP8NotebookBear` for example.

**Comment by AkshJain99** [comment]
what actually i need to do in this issue

**Comment by jayvdb** [comment]
"Update nested language proposal" - like the title says

**Comment by AkshJain99** [comment]
Yes but what type of language proposal i need to do, and what type type of updates i should do in the proposal???

**Comment by AkshJain99** [comment]
@jayvdb There can many examples for the combinations of nested language do i need to update all the combinations or is there any limit????

**Comment by AkshJain99** [comment]
@jayvdb can u tell me what kind of updates should I need to do in proposal

**Comment by jayvdb** [comment]
@AkshJain99 , do you best effort!
